pkg/client/clientset.go

```diff
@@ -179,6 +171,11 @@ func CreatePlatformClientSet(url string, auth PlatformAuth, opts ClientOptions)
 		return nil, err
 	}
 
+	autClient, err := clientFactory.AutomationClient()
```

Comment:
setup is reused from the clientFactory built for the bucket client, which is nice and simple 👌